guides/common/modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc

```diff
@@ -116,62 +104,29 @@ You can find the log for {Project} in `/var/log/foreman/production.log`.
 * `/var/log/httpd/foreman_ssl_access.log`
 
 .Procedure
-. Set the logging level in `/etc/foreman/settings.yaml`:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-:logging:
-  :production:
-    :type: file
-    :layout: pattern
-    :level: debug
-----
-. Enable selected loggers in `/etc/foreman/settings.yaml`:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-:loggers:
-  :ldap:
-    :enabled: true
-  :permissions:
-    :enabled: true
-  :sql:
-    :enabled: true
-----
-+
-Note that to see logging from some area, debug logging has to be set.
-. Restart {Project} services:
+. Set the logging level:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-maintain} service restart
+{foreman-installer} --foreman-logging-level debug \
+--foreman-loggers ldap:true \
+--foreman-loggers permissions:true \
+--foreman-loggers sql:true
 ----
 
-You can find the complete list of loggers with their default values in `/usr/share/foreman/config/application.rb` in the `Foreman::Logging.add_loggers` command.
+TODO: this is a combination of proc_enabling-individual-loggers.adoc and guides/common/modules/proc_enabling-debug-logging.adoc
```

Comment:
I would in this case delete the section and include aforementioned modules in the assembly instead. Feel free to also make changes to the modules wrapped in `ifdef`s if they require slight changes in this particular context.

Reply:
I'd reword this to "You can find the complete list of loggers with their default values in `/usr/share/foreman/config/application.rb` under `Foreman::Logging.add_loggers`."

Reply:
To be clear, `proc_enabling-individual-loggers.adoc` tells you effectively about `--foreman-loggers` and how to use it while `guides/common/modules/proc_enabling-debug-logging.adoc` tells you to use `--foreman-logging-level debug`. This section is essentially telling the users to use both at the same time.